tools/external_ic.F90

```diff
@@ -196,8 +196,8 @@ module external_ic_mod
    real, parameter:: zvir = rvgas/rdgas - 1.
    real(kind=R_GRID), parameter :: cnst_0p20=0.20d0
    real, parameter :: deg2rad = pi/180.
-   character (len = 80),public :: source   ! This tells what the input source was for the data
-   character(len=27), parameter :: source_fv3gfs = 'FV3GFS GAUSSIAN NEMSIO FILE'
+   character(len=80), public:: source
+   character(len=26), parameter:: data_source_fv3gfs='FV3GFS NEMSIO/NETCDF/GRIB2'
```

Comment:
parameter with the same name (`data_source_fv3gfs`) and the same value (`'FV3GFS NEMSIO/NETCDF/GRIB2'`) is already defined in `model/fv_regional_bc.F90`. It is difficult to maintain this and make sure the value is exactly the same in both places. Can't you declare the one in fv_regional_bc as public and use it here?

Reply:
@DusanJovic-NOAA, I've changed the string expression to the logical flag (data_source_fv3gfs) as @lharris4 recommended above. It makes it easy to maintain two files because the input source strings are controlled only in the subroutine 'get_data_source'.